src/frontend-admin/src/components/App/App.module.css

```diff
@@ -34,34 +35,70 @@
   letter-spacing: -1.38px;
 }
 
+.navEllipse {
+  position: relative;
+  left: 0px;
+  background: #333;
+  width: 40px;
+  height: 30px;
+  border-radius: 50%;
+  z-index: 3;
+}
+
+.navRectangle {
+  background: #f4f4f4;
+  position: relative;
+  top: 15px;
+  left: 40px;
+  width: 50px;
+  height: 20px;
+  border-radius: 0px 2px 0px 0px;
+  z-index: 3;
+}
+
+.navTabRectangle {
+  background: #333;
+  position: relative;
+  left: -50px;
+  width: 20px;
+  height: 20px;
+  border-radius:  0 0px 0px -18px; 
+}
+
 .linkContainer {
   align-self: flex-end;
   margin-right: 60px;
-  background: #333;
   border-radius: 0 18px 18px 18px; 
   display: flex;
   align-content: center;
   height: 80px;
+  flex-direction: row-reverse;
 }
 
 .navTab {
-  width: 120px;
+  width: 123px;
   padding: 0px 35px 5px 35px;
   color: #fff;
   text-decoration:none;
   font-size: 15px;
   line-height: 18px;
   background: #333;
-  border-radius: 0 18px 0px 18px;
+  border-radius: 0 18px 0px 0px;
   display: flex;
+  z-index: 3;
+}
+
+.navTab:first-child {
+  border-radius: 0 18px 18px 0;
 }
 
-.navTab:last-Child {
-  border-radius: 0 18px 18px 18px;
+.navTab:nth-child(5) {
```

Comment:
It's ok to use this selector, but it's important to be cautious about possible future updates. If an additional navTab is added in the middle somewhere, will this break? `first-child` and `last-child` tend to be more robust to changes like that.